src/components/ScValPrettyJson/index.tsx

```diff
@@ -31,7 +33,9 @@ export const ScValPrettyJson = ({
 
   const parseJson = () => {
     try {
-      return parse(StellarXdr.decode("ScVal", xdrString)) as AnyObject;
+      return xdrString
+        ? (parse(StellarXdr.decode("ScVal", xdrString)) as AnyObject)
```

Comment:
👁️ I should adopt this `parse` from `lossless json` in `getLedgerEntries`.

Reply:
Yes, we should use this instead of the native `JSON.parse` when dealing with large numbers.